File: apps/mobile/modules/rishi-pdf-extractor/android/src/main/java/expo/modules/rishipdfextractor/RishiPdfExtractorModule.kt
package expo.modules.rishipdfextractor

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class RishiPdfExtractorModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("RishiPdfExtractor")

    Function("hello") {
      "hello from rishi-pdf-extractor (android)"
    }
  }
}

File: apps/mobile/modules/rishi-pdf-extractor/ios/RishiPdfExtractorModule.swift
import ExpoModulesCore

public class RishiPdfExtractorModule: Module {
  public func definition() -> ModuleDefinition {
    Name("RishiPdfExtractor")

    Function("hello") { () -> String in
      "hello from rishi-pdf-extractor (ios)"
    }
  }
}
